src/commands/config/set.js

```diff
@@ -135,21 +128,50 @@ const handlers = {
 	pretty: setBoolean,
 };
 
-export const actionCreator = () => async ({ variable, values }) => {
-	if (!CONFIG_VARIABLES.includes(variable)) {
-		throw new ValidationError('Unsupported variable name.');
+export default class SetCommand extends BaseCommand {
+	async run() {
+		const { args: { variable, values } } = this.parse(SetCommand);
+		const safeValues = values || [];
+		const safeValue = safeValues[0] || '';
+		const result = handlers[variable](
+			this.userConfig,
+			variable,
+			safeValue,
+			safeValues,
+		);
+		this.print(result);
 	}
-	const safeValues = values || [];
-	const safeValue = safeValues[0] || '';
-	return handlers[variable](variable, safeValue, safeValues);
+}
+
+SetCommand.args = [
+	{
+		name: 'variable',
+		required: true,
+		options: CONFIG_VARIABLES,
+		description: '',
+	},
+	{
+		name: 'values',
+		required: false,
+		parse: input => input.split(','),
+		description: '',
+	},
+];
+
+SetCommand.flags = {
+	...BaseCommand.flags,
 };
 
-const set = createCommand({
-	command: 'set <variable> [values...]',
-	autocomplete: CONFIG_VARIABLES,
-	description,
-	actionCreator,
-	errorPrefix: 'Could not set config variable',
-});
+SetCommand.description = `
+Sets configuration <variable> to [value(s)...].
+...
+Variables available: ${availableVariables}. Configuration is persisted in \`${
+	process.env.XDG_CONFIG_HOME
```

Comment:
```
./bin/run config:show                                                                                                                                                  
{"name":"Lisk Commander","delimiter":"lisk","json":true,"api":{"nodes":[],"network":"main"},"pretty":false}
```
Mine seems to be disagree...

Reply:
Sorry meant `./bin/run help config:show`

Reply:
```
./bin/run help config:show                                                                                                                                             
USAGE
  $ lisk config:show

OPTIONS
  -j, --json  Prints output in JSON format. You can change the default behaviour in your config.json file.
  --pretty    Prints JSON in pretty format rather than condensed. Has no effect if the output is set to table. You can change the default behaviour in your config.json file.

DESCRIPTION
  Prints the current configuration.

EXAMPLE
  config:show
```

Reply:
Eesh, I guess I meant `./bin/run config`. See output in next comment.